File: Daily Challanges/2026-06-June/2026-06-24.py
def detect_mutations(strand1, strand2):
    return [i for i in range(len(strand1)) if strand1[i] != strand2[i]]


print(detect_mutations("ATCG", "ATGG"))
print(detect_mutations("ATGCGTACGTTAGC", "ATGCATACGATTGC"))
print(detect_mutations("GATCTAGCTAGGCTAGCTAG", "GATCTAGCTAGGCTAGCTAG"))

File: Daily Challanges/2026-06-June/2026-06-24.js
function detectMutations(strand1, strand2) {
  let mutations = [];
  for (let i = 0; i < strand1.length; i++) {
    if (strand1[i] != strand2[i]) mutations.push(i);
  }
  return mutations;
}

console.log(detectMutations("ATCG", "ATGG"));
console.log(detectMutations("ATGCGTACGTTAGC", "ATGCATACGATTGC"));
console.log(detectMutations("GATCTAGCTAGGCTAGCTAG", "GATCTAGCTAGGCTAGCTAG"));
